### PR TITLE
Created pets model router

### PR DESCRIPTION
**Pull request by RobertElias**
Created pets router and model.

### CODE REVIEW

File: __tests__/petstest.test.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    it('should return 200 when pets is created', async () => {
+      const pet = {
+        name: 'Chuckee',
+        description: 'Pug breed with no medical conditions',
+        avatarUrl:
+          'https://s3.amazonaws.com/uifaces/faces/twitter/hermanobrother/128.jpg',
+      };
+      Pets.findById.mockResolvedValue(undefined);
+      Pets.create.mockResolvedValue([
+        Object.assign({ id: 'd376de0577681ca93614' }, pets),
```
Comment (coryortega) [inline __tests__/petstest.test.js:65]: "pets" should be "pet"

File: __tests__/petstest.test.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      ]);
+      const res = await request(server).post('/pets').send(pets);
```
Comment (coryortega) [inline __tests__/petstest.test.js:67]: "pets" needs to be "pet"

**Review by coryortega — CHANGES_REQUESTED** [review]
Minor change needs to happen, like using "pet" instead of "pets" and test coverage needs to be adjusted

File: __tests__/petstest.test.js (hunk 1 above)
Comment (RobertElias) [reply __tests__/petstest.test.js]: updated pets to pet and changed file name to pets.test.js

**Review by williamschwindt — APPROVED** [review]
Awesome job Robert! Looks like a few ESLint/prettier problems but other then that nice work.

**Pull request closed without merging**